commit a32c51ad703da60a5f42d98422e22c160bede1ac
Author: Vitaly Kuznetsov <vkuznets@redhat.com>
Date:   Fri Mar 17 00:48:26 2017 +0000

    Drivers: hv: ring_buffer: count on wrap around mappings in get_next_pkt_raw() (v2)
    
    [ Upstream commit fa32ff6576623616c1751562edaed8c164ca5199 ]
    
    With wrap around mappings in place we can always provide drivers with
    direct links to packets on the ring buffer, even when they wrap around.
    Do the required updates to get_next_pkt_raw()/put_pkt_raw()
    
    The first version of this commit was reverted (65a532f3d50a) to deal with
    cross-tree merge issues which are (hopefully) resolved now.
    
    Signed-off-by: Vitaly Kuznetsov <vkuznets@redhat.com>
    Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
    Tested-by: Dexuan Cui <decui@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1432_a32c51ad703da60a5f42d98422e22c160bede1ac.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1432_a32c51ad703da60a5f42d98422e22c160bede1ac.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
